svir/dialogs/viewer_dock.py

```diff
@@ -29,13 +29,18 @@
 import io
 from collections import OrderedDict
 
-from matplotlib.figure import Figure
-from matplotlib.backends.backend_qt4agg import (
-    FigureCanvasQTAgg as FigureCanvas,
-    NavigationToolbar2QT as NavigationToolbar
-)
-from matplotlib.lines import Line2D
-
+try:
+    from matplotlib.figure import Figure
+    from matplotlib.backends.backend_qt4agg import (
+        FigureCanvasQTAgg as FigureCanvas,
+        NavigationToolbar2QT as NavigationToolbar
+    )
+    from matplotlib.lines import Line2D
+except ImportError as exc:
+    raise ImportError(
+        'There was a problem importing matplotlib. It could happen when using'
```

Comment:
Minor EN suggestion:
"There was a problem importing matplotlib, if you are using a 64bit version of QGIS with on Windows, please try using a 32bit version instead."